test/test_ccf.py

```diff
@@ -141,3 +141,39 @@ def submit(**kwargs):
         submit(alg="ES256", kty="ec", ec_curve="P-256")
         submit(alg="ES384", kty="ec", ec_curve="P-384")
         submit(alg="PS256", kty="rsa")
+
+
+def test_accepted_did_issuers(tmp_path: Path):
```

Comment:
This could probably be moved in a new `test_configuration.py` file instead. If you want, also move the `test_accepted_algorithms` into that file at the same time (it's fine if you would rather leave this for later)